android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -97,8 +99,12 @@
     public static final int USER_WALLET_ACTIVITY_REQUEST_CODE = 35;
     public static final String ADD_FUNDS_URL = "chrome://rewards/#add-funds";
     public static final String REWARDS_SETTINGS_URL = "chrome://rewards/";
+    public static final String BRAVE_REWARDS_SETTINGS_URL = "brave://rewards/";
```

Comment:
do we need that one and one above duplicated?

Reply:
@SergeyZhukovsky to show a custom dialog in rewards settings page, we need to check for both urls as you can go to the same page by same url. so i had to add it here

Reply:
sounds good